### ISSUE TITLE
[Release Partner] Tabrasa

### ISSUE THREAD

**Issue by brucejones25** [carbon-design-system/carbon#8692]
**What is your team's name?**

Tabrasa

**Why would you like to join the Release Partner Program?**

We love Carbon and, coincidentally, we're just starting to replace the UI for out entire platform with a new React front end.  Oh, and I worked at IBM for 24 years and I'm still a big fan!

**Are you interested in participating in biweekly feedback sessions for the release?**

Absolutely - we'd love to.  

**Who should we reach out to for more info?**

bruce@tabrasa.io

**Comment by joshblack** [comment]
Hi there, @brucejones25! 👋 

Thanks so much for reaching out! This is incredibly exciting. We're hoping to open up this program to all teams starting with our next pre-release in the next couple of weeks, just need to figure out the logistics.

While you wait, you can follow along with our updates that are being posted here: https://github.com/carbon-design-system/carbon/blob/main/docs/guides/release-partners.md We just added one for May 13th which is the latest update.

We're definitely still in a rough period for the release, things are very much alpha/beta quality, but you're more than welcome to check it out. As we start moving towards our first pre-releases then you can expect to hear back from me regarding the schedule, how to try it out, how to give feedback, and all that good stuff.

Hope that helps! Let me know if you have any questions, and thanks again for reaching out! We're lucky to have you and your team 😄

**Comment by joshblack** [comment]
Hi there! Just wanted to follow up on this to share that we created a space for release partners over on [Discord](https://discord.gg/5tDVQH98). If you let me know your Discord username over in that space, I can add the "Release Partner" role to your account and you'll see some of the channels that we have set up for this process.

Let me know if you have any questions! We're so happy that you reached out about this and excited to work with you 😄
